COPY/usr/bin/manageiq-messaging-ready

```diff
@@ -42,10 +42,33 @@ else
   true
 end
 
+def initialize_topics(msg_yaml, env = "production")
+  require "shellwords"
+
+  host, port = msg_yaml[env].values_at("host", "port")
+  host ||= "127.0.0.1"
+  port ||= 9092
+  bootstrap_server = Shellwords.shellescape("#{host}:#{port}")
+
+  `/opt/kafka/bin/kafka-topics.sh --create --topic manageiq.ems --bootstrap-server #{bootstrap_server} --command-config /opt/kafka/config/client.properties`
+  `/opt/kafka/bin/kafka-topics.sh --create --topic manageiq.ems-events --bootstrap-server #{bootstrap_server} --command-config /opt/kafka/config/client.properties`
+  `/opt/kafka/bin/kafka-topics.sh --create --topic manageiq.ems-inventory --bootstrap-server #{bootstrap_server} --command-config /opt/kafka/config/client.properties`
+  `/opt/kafka/bin/kafka-topics.sh --create --topic manageiq.metrics --bootstrap-server #{bootstrap_server} --command-config /opt/kafka/config/client.properties`
```

Comment:
On my machine (Mac), I have this as a binary on the path and not a .sh file...I'm wondering if it's also on the path in appliance, then we don't need the explicit path.

```sh
$ which kafka-topics
/opt/homebrew/bin/kafka-topics
```

Reply:
This is what I see on a downstream appliance:
```
$ which kafka-topics
/usr/bin/which: no kafka-topics in (/opt/IBM/infrastructure-management-gemset/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/root/bin)
```

Reply:
Ok, I'll merge as is - I still think there might be a nicer way to do this, but this works for now.

Reply:
Yeah in the linux kafka package there is only `kafka-topics.sh` which has a `#!/bin/bash` shebang and is executable so the `.sh` isn't really necessary.